README.md

```diff
@@ -138,6 +138,7 @@ The lists are displayed in an interactive UI by default.
 - Use arrow keys or `j, k, h, l` characters to navigate through the list.
 - Use `g` and `SHIFT+G` to quickly navigate to the top and bottom respectively.
 - Press `v` to view selected issue details.
+- Press `r` to refresh issues list.
```

Comment:
Nitpick, this is still pointing to previous refresh shortcut.

Reply:
yes of course! Sorry I missed that!